source/agg-pipeline-builder/maxtime-ms-pipeline.txt

```diff
@@ -0,0 +1,65 @@
+.. _set-maxtime-ms-agg:
+
+=====================================
+Set maxTimeMS for Aggregation Queries
+=====================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Use the ``maxTimeMS`` parameter on the :guilabel:`Aggregations` tab to 
+specify an upper time limit in milliseconds for aggregation pipelines
+that run in |compass|.
+
+About This Task 
+---------------
+
+If you already specified a ``maxTimeMS`` value in the :ref:`Compass Settings 
```

Comment:
[question] What do you think about the placement of this note? I originally had it as a note admonition under step 2, but felt it'd be more helpful here to know beforehand.